draft-ietf-tls-tls13.md

```diff
@@ -5846,6 +5890,31 @@ then erase the \[early_]exporter_master_secret, followed by
 each inner values as soon as it is known that it will not be
 needed again.
 
+### Cross-Protocol Private Key Reuse
+
+Use of the same private key for different versions of TLS exposes that
+key to attacks in all versions of the protocol, as demonstrated by
+the {{DROWN}} attack. Previous versions of TLS were further susceptible
+to cross-protocol attacks between different ciphersuites {{WAGNER}}.
+This version of TLS takes steps to thwart cross-protocol attacks between
+different ciphersuites, and between different protocol versions sharing
+the same private key (see {{certificate-verify}}) for the same purpose.
+
+However, it does not mandate the separation of the private key and
+certificate for different protocol versions. It is, possible,
+to reuse keys for PKCS#1 1.5 RSA decryption under TLS 1.2,
+for RSASSA-PSS signatures under TLS 1.3, enabling attacks
+similar to {{DROWN}} to affect TLS 1.3 deployments.
+For that reason, it is RECOMMENDED to
+
+- Use separate keys between different protocol versions;
+- Use an RSASSA-PSS-only key with its corresponding certificate for TLS 1.3.
```

Comment:
"Use a key with public key ID of RSASSA-PSS for TLS 1.3" ?

Reply:
I do not think we can easily make it more specific; the subject public key identifier which I believe you refer to is only on certificates. PKCS#8 private keys use the algorithmIdentifier field. I think it is better to keep it generic as there are quite few private key types, and what could an RSASSA-PSS-only key may mean on each will vary.